src/theme/ThemeProvider.tsx

```diff
@@ -29,6 +29,13 @@ export const ThemeProvider: FC<ThemeProviderProps> = ({ children }) => {
           fontSize: '20px',
           fontFamily: 'system-ui, -apple-system, BlinkMacSystemFont'
         }
+      },
+      MuiAppBar: {
```

Comment:
Instead of overriding, the app bar should probably be the main color. The point of the theme provider is to provide consistency across the application, so overrides break that.

[MUI Theming](https://mui.com/material-ui/customization/theming/)